plyer/platforms/linux/wifi.py

```diff
@@ -17,7 +17,10 @@
 
 class LinuxWifi(Wifi):
     names = {}
-
+    
+    Popen(["nmcli", "-t","connection", "show", "--active"], stdout=PIPE, stderr=PIPE).
+    ifname = communicate()[0].decode('utf-8').split(":")[-1].strip()
```

Comment:
While this seems nice in general, `communicate()` is a method of an object returned from `Popen()` and you call it out of nowhere.

Run your code first before committing and also include a default `wlan0` value in case the `Popen()` call fails with non-zero error code and/or invalid value because the original implementation uses old arguments in the `call()`, therefore your way of fetching the interface might not be available in the old `nmcli` binary.

Reply:
Sorry, i was on my tablet so, i commit directly. :p